docs/data-sources/builtin-toolsets/splunk-mcp.md

```
@@ -0,0 +1,164 @@
+    helm upgrade --install robusta robusta/robusta -f generated_values.yaml --set clusterName=YOUR_CLUSTER_NAME
+    ```
+
+## Available Tools
+
+The Splunk MCP server provides tools for searching and analyzing data in Splunk. For the complete list of available tools and their parameters, see the [Splunk MCP Server Tools documentation](https://help.splunk.com/en/splunk-cloud-platform/mcp-server-for-splunk-platform/mcp-server-tools).
+
+## Testing the Connection
+
+After configuring Holmes to connect to Splunk MCP, verify it's working:
+
+```bash
```

markdownlint-cli2 (0.20.0)

Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)